src/pubsub/unsubscribe.js

```diff
@@ -57,5 +57,21 @@ module.exports = (createCommon, options) => {
         )
       })
     })
+    it('should subscribe and unsubscribe with no handler', (done) => {
+      const someTopic = getTopic()
+      const handler = (msg) => {}
+      ipfs.pubsub.subscribe(someTopic, handler, (err) => {
```

Comment:
It would be good to add more than one handler for `someTopic` so we're verifying it is indeed removing everything.

Reply:
done